scenarios/workflow_on_many_task_queues.go

```diff
@@ -0,0 +1,33 @@
+package scenarios
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/temporalio/omes/loadgen"
+	"github.com/temporalio/omes/loadgen/kitchensink"
+)
+
+func init() {
+	loadgen.MustRegisterScenario(loadgen.Scenario{
+		Description: "Each iteration executes a single workflow on one of the task queues. " +
+			"Workers must be started with --task-queue-suffix-index-end as one less than task queue count here. " +
+			"Additional options: task-queue-count (required).",
+		Executor: loadgen.KitchenSinkExecutor{
+			WorkflowParams: kitchensink.NewWorkflowParams(kitchensink.NopActionExecuteActivity),
+			PrepareWorkflowParams: func(ctx context.Context, opts loadgen.RunOptions, params *kitchensink.WorkflowParams) error {
+				// Require task queue count
+				if opts.ScenarioOptionInt("task-queue-count", 0) == 0 {
```

Comment:
This works, though I wish it was more tightly coupled to the run options and less to reduce the chance of human error.

Reply:
Yeah, I could use https://pkg.go.dev/github.com/spf13/pflag on a per scenario basis, but it's hard to use with the command because you have to know the scenario first. You can tell it "all flags are valid" and just parse unparsed ones after you've resolved the scenario, but that's a bit ugly. Yes, I could make a full argument thing on my own with data types and such, but it seemed like overkill though I wouldn't be against it.